.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "node_modules\\typescript\\lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
+}
```

Comment:
Why do we need this?

Reply:
This is by mistake , ill remove it in the next PR